test/Swashbuckle.AspNetCore.IntegrationTests/SwaggerIntegrationTests.cs

```diff
@@ -31,39 +31,45 @@ public class SwaggerIntegrationTests
             string swaggerRequestUri)
         {
             var testSite = new TestSite(startupType);
-            var client = testSite.BuildClient();
+            using (var client = testSite.BuildClient())
```

Comment:
Can you change these all to use `using var client = testSite.BuildClient();` instead please?

It's newer syntax that avoids all these indentation changes where the scope ends at the end of the method anyway, so it's the same.

At the same time, all the responses can also be made `using var` as `HttpResponseMessage` implements `IDisposable` too.